src/styles/main.scss

```diff
@@ -1 +1,11 @@
-@import './base/index.scss'
+@import './base/index.scss'; 
+@import './layout/index.scss';
+@import './modules/index.scss';
+@import './state/index.scss';
+@import './theme/index.scss';
+@import './modules/nav-list-iteam-text.scss';
```

Comment:
you already have `modules/index.scss` import, your main modules import should contains all your submodules and be imported in `main.scss`. You have to move your modules imports to `modules/index.scss`

Reply:
done